preprocess/analysis/handlers/test_routes.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Jul 20 16:36:57 2020
+
+@author: scarlet
+"""
+from flask import Flask
+from routes import configure_routes, MAX_CHAR_LIMIT
+from test_language_analysis import STRING_ONE, STRING_TWO
```

Comment:
Just put test strings in this test, importing test cases from another test makes your tests dependent on each other and is overly complicated.